src/theme/components/checkbox.ts

```diff
@@ -87,7 +87,7 @@ const sizes = {
     // Controls the width/height of the checkbox itself.
     control: {
       ...checkboxRadioControlSize,
-      borderRadius: "sm",
+      borderRadius: "3px",
```

Comment:
We can create custom radius values. Maybe this should be a custom name as well.

- Here's the theme file: https://github.com/NYPL/nypl-design-system/blob/development/src/theme/foundations/radii.ts
- And how it's used: https://github.com/NYPL/nypl-design-system/blob/development/src/theme/components/button.ts#L67

Reply:
Good point.  I made the change to use a custom radius value.